dolweb/downloads/views.py

```diff
@@ -40,6 +40,12 @@ def branches(request):
 
     return { 'branches': branches }
 
+@render_to('downloads-buildlist.html')
```

Comment:
Don't go through a template for such a simple thing. Ideally use instead a [JsonResponse](https://docs.djangoproject.com/en/1.10/ref/request-response/#jsonresponse-objects) with e.g. {'versions': ['a', 'b', 'c', ...]}.